keadm/cmd/keadm/app/cmd/common/types.go

```diff
@@ -92,8 +93,8 @@ type ToolsInstaller interface {
 type OSTypeInstaller interface {
 	InstallMQTT() error
 	IsK8SComponentInstalled(string, string) error
-	InstallKubeEdge(ComponentType) error
 	SetKubeEdgeVersion(version semver.Version)
+	InstallKubeEdge(ComponentType, string) error
```

Comment:
add parameter name `packagePath`
better not to change interface definaion

Reply:
+1

Reply:
well, I think I have to add a parameter here...

Reply:
I ss, how about using a struct here and cover component type, package path and something else.